io/net/infiniband/ib_pingpong.py

```diff
@@ -0,0 +1,148 @@
+#!/usr/bin/env python
+
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# Copyright: 2016 IBM
+# Author: Prudhvi Miryala<mprudhvi@linux.vnet.ibm.com>
+# ping pong test
+# ibv_ud_pingpong
+# ibv_uc_pingpong
+# ibv_rc_pingpong
+# ibv_srq_pingpong
+
+
+import time
+import netifaces
+from avocado import Test
+from avocado.utils.software_manager import SoftwareManager
+from avocado.utils import process
+from avocado.utils import distro
+
+
+class pingpong(Test):
+    '''
+       ib_pingpong test
+       pingpong tools should be installed
+    '''
+    def setUp(self):
+        '''
+           To check and install dependencies for the test
+        '''
+        sm = SoftwareManager()
+        detected_distro = distro.detect()
+        depends = ["openssh-clients"]
+        if detected_distro.name == "Ubuntu":
+            depends.append("ibverbs")
+        elif detected_distro.name == "redhat":
+            depends.append("libibverbs")
+        for package in depends:
+            if not sm.check_installed(package):
```

Comment:
@PraveenPenguin It is an ISO, but not a repo. Installing MOFED is a prerequisite for any IB tests.